app/command_invite.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I think the latest agreement is just to set this to "2015-present"

Reply:
ahh. I did not know that. will update. thanks for the review!

Reply:
done